extensions/github/package.json

```diff
@@ -193,6 +194,22 @@
     }
   ],
   "preferences": [
+    {
```

Comment:
Could you move both preferences in the `Search Repositories` command preferences? That way it doesn't pollute the global preferences for these not using it.